packages/beacon-node/src/chain/archiveStore/utils/archiveBlocks.ts

```diff
@@ -112,9 +112,29 @@ export async function archiveBlocks(
     }
   }
 
+  // Delete expired data column sidecars
+  // Keep only `[current_epoch - max(MIN_EPOCHS_FOR_DATA_COLUMN_SIDECARS_REQUESTS, archiveBlobEpochs)]`
   if (finalizedPostFulu) {
-    // TODO
-    // Keep only `[current_epoch - max(MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS, archiveBlobEpochs)]
+    if (archiveBlobEpochs !== Infinity) {
```

Comment:
Do you mind if we change `archiveBlobEpochs` to `archiveDataEpochs` back to the config/flag that sets the value?